index.mjs

```diff
@@ -0,0 +1,35 @@
+import JsoncParser from './lib/umd/main.js';
```

Comment:
can this be lib/esm/main.js?

Reply:
No, in `lib/esm/main.js`, we import files without `'.js'` suffix, which will cause error.

Reply:
https://nodejs.org/api/esm.html#esm_terminology

> Relative specifiers like './startup.js' or '../config.mjs'. They refer to a path relative to the location of the importing file.